pkg/recommender.py

```diff
@@ -83,9 +93,22 @@ def update(self, *args, **kargs):
 
 
 class Recommender:
-    def __init__(self, evt_dim=5, mock=False, server_config=temp_server_config, mode='default'):
+    def __init__(self, evt_dim=5, mock=False, server_config=temp_server_config, 
+    mode='default', test=False, time_config=None, schedule_evt_test_config=None):
```

Comment:
I feel as long as the interface of the constructor and function `dispatch` stays the same, we do not need to inform acoustic side to have any updates. Acoustic system should use all these default arguments, right?

Reply:
Oh okay and yes I believe so